calculator.py

```
@@ -16,23 +16,100 @@ def divide(a, b):
+
+        if name.lower() == 'выход':
+            break
+
+        gender = guess_gender_by_name(name)
+
+        # Добавляем описание
+        descriptions = {
+            "Мужской": "традиционная мужская идентичность",
+            "Женский": "традиционная женская идентичность",
+            "Небинарный": "идентичность вне бинарной системы мужчина/женщина",
+            "Гендерфлюид": "гендер изменяется со временем",
```

Ruff (0.13.2)

String contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF001)